.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug
+about: Create a bug report to help us improve
+title: "[BUG] Feature name - Short description of the bug"
+labels: ''
+assignees: ''
+
+---
+
+_(Feel free to write you report in French if you prefer.)_
+
+## Required information
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+If you fond another bug report related to this issue on GitHub, please mention its number e.g. #7890
```

Comment:
Change "fond" to "find".

Reply:
On second thought, was it supposed to be "find" or "found"?

Reply:
found. Thanks!